src/main/kotlin/test/ktortemplate/conf/database/DatabaseConnection.kt

```diff
@@ -23,4 +26,15 @@ class DatabaseConnection(private val dataSource: DataSource) {
     ): T = newSuspendedTransaction(Dispatchers.IO, database, txIsolation) {
         block()
     }
+
+    suspend fun <T> suspendedQueryContext(
```

Comment:
If we want every log to print the request id inside coroutines, we have to pass the MDC context when creating a new coroutine. I'm not sure if this is the correct way to do this, so I created a new function so we can discuss this. @monxalo @felix19350 do you know another way to do this? Because the `newSuspendedTransaction` function is expecting a `CoroutineDispatcher` instead of a `CoroutineContext` so we can't use the plus operator `coroutineContext + Context()`.

Reply:
`newSuspendedTransaction` has that implementation, which is unfortunate since the respective implementation it passes that value to function accepting a `CoroutineContext`.

Reply:
I just merged the `suspendedQueryContext` with the `suspendedQuery`, I'm not a big fan of giving the users too many ways to do the same thing.